### PR TITLE
OBSDOCS-1062 - Logging 5.8.7 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.8.7
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1062

Fix Version: 4.12, 4.13, 4.14, 4.15

Doc Preview: https://76276--ocpdocs-pr.netlify.app/openshift-dedicated/latest/observability/logging/logging_release_notes/logging-5-8-release-notes.html#logging-release-notes-5-8-7_logging-5-8-release-notes

SME Review: @periklis 
QE Review: @anpingli 
Peer Review: @stevsmit

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

**Comment by stevsmit** [comment]
Just a few nits!

File: modules/logging-release-notes-5-8-7.adoc [hunk 1]
```diff
@@ -0,0 +1,118 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-7_{context}"]
+= Logging 5.8.7
+This release includes link:https://access.redhat.com/errata/RHSA-2024:2932[OpenShift Logging Bug Fix Release 5.8.7 Security Update] and link:https://access.redhat.com/errata/RHBA-2024:2931[OpenShift Logging Bug Fix Release 5.8.7].
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-8-7.adoc:5]: That's correct. the links will work fine once the release is live.

**Pull request merged**